project/zadanie_domowe/ex3.py

```diff
@@ -0,0 +1,16 @@
+# 3a. czy liczba jest podzielna przez 3 lub 5 lub 7
+# 3b. czy liczba jes podzielne przez 3 i 5 i 7
+
+liczba = int(input("Wprowadź swoją ulubioną liczbę: "))
+
+if liczba % 3 == 0 or liczba % 5 == 0 or liczba % 7 == 0:
+    print(f"Twoja liczba jest podzielna przez 3,5 lub 7.")
```

Comment:
jak to jest moja ulubiona liczba to przez które z tych liczb jest ona podzielna konkretnie? ;)